### PR TITLE
Create new Architecture and Styling blocks

### PR DESCRIPTION
**Pull request by roke77**

### CODE REVIEW

**Review by abelosorio — COMMENTED** [review]
So far so good.

File: content/block-2/index.md [hunk 1]
```diff
@@ -20,9 +20,18 @@
 ### React
 
 - Tutorial by React: https://reactjs.org/tutorial/tutorial.html (use the *Local environment setup*).
-- If you have time, take a look at the [Wrapping up](https://reactjs.org/tutorial/tutorial.html#wrapping-up) section.
+- React Hooks: https://reactjs.org/docs/hooks-intro.html
 
-## Wrap-up
+### Github ??? (maybe a better title like: Collaboration)
```
Comment (nahudalla) [inline content/block-2/index.md]:
```suggestion
### Git Flow in practice
```

File: content/block-2/index.md [hunk 2]
```diff
@@ -20,9 +20,18 @@
...
+
+- TODO: How to create a Pull Request: https://
```
Comment (nahudalla) [inline content/block-2/index.md]:
```suggestion
- How to create a Pull Request: https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request
```

File: content/block-2/index.md [hunk 3]
```diff
@@ -20,9 +20,18 @@
...
+- How to Write a Git Commit Message: https://chris.beams.io/posts/git-commit/
+
+## Tasks
+
+1. Create a public repository into your Github account to host the game example from the tutorial. Make sure to put in practice the [Git Flow](https://datasift.github.io/gitflow/IntroducingGitFlow.html).
+2. Create a new Pull Request extracting components into its own files (TODO: FE Arch ???) and converting React Class Components into Functional Components.
```
Comment (nahudalla) [inline content/block-2/index.md]: FE architecture is not the focus of this block. They'll be doing that in B4.
```suggestion
2. Create a new Pull Request extracting components into its own files and converting React Class Components into Functional Components.
```

File: content/block-3/index.md [hunk 4]
```diff
@@ -1,56 +1,42 @@
...
 
-# Block 3: Start sample project (2 weeks)
+# Block 3: Design (2 weeks)
 
 ## Contents
-
-- GraphQL.
-- ORM (Sequelize).
+- Component Diagrams
+- Advanced CSS
```
Comment (nahudalla) [inline content/block-3/index.md]: 🤔 the focus of this block should be only `Component Diagrams` and `Team Collaboration`. CSS and FE Architecture is Block 4.

**Review by nahudalla — CHANGES_REQUESTED** [review]

**Review by MPucheta — APPROVED** [review]

**Review by nahudalla — APPROVED** [review]

**Pull request merged**